frontend-cert/js-projects/dates-lab/script.js

```diff
@@ -0,0 +1,21 @@
+const currentDate = new Date();
+console.log('Current Date and Time:', currentDate);
```

Comment:
Just like we did earlier with the `shortDateFormat` and `longDateFormat` we should have them create a variable called `currentDateFormat`. There should be a user story similar to the other two.